eth_utils/crypto.py

```diff
@@ -3,13 +3,13 @@
 except ImportError:
     from sha3 import sha3_256 as keccak_256
 
-from .string import (
-    force_bytes,
+from .conversions import (
+    to_bytes,
 )
 
 
 def keccak(value):
-    return keccak_256(force_bytes(value)).digest()
+    return keccak_256(to_bytes(text=value)).digest()
```

Comment:
Is it safe to assume all `value` args come in as text? The tests pass but I wasn't sure if i should convert this to the (primitive, hexstr, text) format as well?

Reply:
No, I don't think this is a safe assumption.  I actually think that this function should be changed to have the same interface of `keccak(primative=None, text=None, hexstr=None)`.